Controller/Payment/Notification.php

```diff
@@ -32,7 +32,9 @@ public function execute()
 
             $this->getResponse()->setBody('ok');
             foreach ($incrementIds as $id) {
-                $order  = $this->paymentOrderRepository->getOrderByIncrementId($id);
+                $entityId = $this->paymentOrderRepository->getOrderByIncrementId($id)->getEntityId();
+                $order  = $this->paymentOrderRepository->getOrderByEntityId($entityId);
+                $order->getEntityId();
```

Comment:
Seems like the return value of `$order->getEntityId();` is not stored in any of the variable. Is this typo or expected to call that function without needing the return value?

Reply:
Unused code @rizdaprasetya, already removed. Thanks